### DISCUSSION TITLE
github copilot teacher approval.

### DISCUSSION THREAD

**Discussion by celikcedev** [community/community#128149] [GitHub Education] [answered]
Hi, I am a teacher. My github copilot application was approved on 09.06.224. When will I be able to access its benefits? I would be happy if you could help me.

**Comment by Akshitha-reddy-13** [comment]
It takes around a minimum of 72 hours, or sometimes even more, I guess. Maybe you should wait and see a little more, sir.

**Reply by celikcedev** [reply]
Thank you. I'll wait  impatience.

**Comment by Uknowme-h** [comment]
i think you will get an email after few hours

**Reply by celikcedev** [reply]
I hope so.

**Comment by bennefeldtim** [comment]
Did you try these steps?

Install the GitHub Copilot Extension:

For Visual Studio Code, you need to install the GitHub Copilot extension. You can do this directly from the Visual Studio Code Marketplace. Here’s how:
Open Visual Studio Code.
Go to the Extensions view by clicking the Extensions icon in the Activity Bar on the side of the window.
Search for "GitHub Copilot".
Click "Install" next to the GitHub Copilot extension.
Activate GitHub Copilot:

After installing the extension, you may need to sign in with your GitHub account to activate GitHub Copilot. Follow the on-screen instructions to complete this process.
Verify Access:

Once installed and signed in, try creating or editing some code. GitHub Copilot should start providing code suggestions.

**Reply by celikcedev** [reply]
Thank you. I'll try it.

**Comment by hoanganh6789** [comment]
Have you received the email yet?

**Reply by celikcedev** [reply]
Not yet.

**Comment by celikcedev** [comment]
I guess I have to wait. The following message appears on the application screen: “Hi, celikcedev! You were last verified as of Jun 9, 2024. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.”

**Comment by samus-aran** [comment, marked as answer]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful and approved - enjoy your journey through GitHub Education.
